src/grid.jl

```diff
@@ -0,0 +1,43 @@
+abstract type Grid{N,T} end
+
+# Free because later: maybe put bounds lb and ub (e.g., for BDDs)
+struct GridFree{N,T} <: Grid{N,T}
+    orig::SVector{N,T}
+    h::SVector{N,T}
+end
+
+function NewGrid(orig::SVector{N,T}, h::SVector{N,T}) where {N,T}
```

Comment:
Why not call directly `GridFree` ? What is the use of `NewGrid` ?

Reply:
Changed